lib/ci-stack.ts

```diff
@@ -28,15 +29,25 @@ export class CIStack extends Stack {
     const useSslParameter = new CfnParameter(this, 'useSsl', {
       description: 'If the jenkins instance should be access via SSL',
       allowedValues: ['true', 'false'],
+      default: 'false',
```

Comment:
I would strongly suggest we do not include default values, default values hide what the deployment is going to do or not.  If we need a way to make it easier, copying/pasting from the readme next to the explanation is a better solution.